From f3d57180bf90cff8cefe16d499497e2df2384fda Mon Sep 17 00:00:00 2001
From: Joerg Hofrichter <joerg.hofrichter@ni.com>
Date: Thu, 2 May 2024 13:47:30 +0200
Subject: [PATCH] mpm: get_mender_artifact: fix obsolete command

"mender -show-artifact" is deprecated since mender version 3.0.0
use "mender show-artifact" instead
---
 mpm/python/usrp_mpm/sys_utils/filesystem_status.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mpm/python/usrp_mpm/sys_utils/filesystem_status.py b/mpm/python/usrp_mpm/sys_utils/filesystem_status.py
index 0a788d8b8..d168d6d84 100644
--- a/mpm/python/usrp_mpm/sys_utils/filesystem_status.py
+++ b/mpm/python/usrp_mpm/sys_utils/filesystem_status.py
@@ -32,7 +32,7 @@ def get_mender_artifact(filesystem_root='/', parse_manually=False):
             return None
         return parse_artifact(file.read_text())
     try:
-        output = subprocess.check_output(['/usr/bin/mender', '-show-artifact']).decode('utf-8')
+        output = subprocess.check_output(['/usr/bin/mender', 'show-artifact']).decode('utf-8')
         return output.splitlines()[0]
     except:
         return None
-- 
2.34.1

